examples/lua/main.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2018 FU Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Basic lua example application
+ *
+ * @author      Daniel Petry <daniel.petry@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include "shell.h"
+
+#include "main.lua.h"
+
+extern int lua_run_char_array(const char *buffer, size_t buffer_len);
```

Comment:
This function should be included from a header file with its proper Doxygen documentation. As far as I see, it's intended to be called by the user